subhub/tests/unit/test_payment_calls.py

```diff
@@ -203,6 +235,17 @@ def test_cancel_subscription_with_valid_data(app, create_subscription_for_proces
     g.subhub_account.remove_from_db("process_test")
 
 
+def test_cancel_subscription_with_invalid_data(app, create_subscription_for_processing):
+    (subscription, code) = create_subscription_for_processing
+    (cancelled, code) = payments.cancel_subscription(
+        "process_test", subscription["subscriptions"][0]["subscription_id"]+"a"
```

Comment:
single char "a" is somewhat ambiguous; why not +"invalid" to make it more explicit?

Reply:
Good suggestion.